official/transformer/v2/transformer_benchmark.py

```diff
@@ -150,6 +151,54 @@ def benchmark_1_gpu(self):
                                    bleu_min=25.3,
                                    bleu_max=26)
 
+  def benchmark_8_gpu(self):
+    """Benchmark 8 gpu.
+
+      The paper uses 8 GPUs and a much larger effective batch size, this is will
+      not converge to the 27.3 BLEU (uncased) SOTA.
+    """
+    self._setup()
+    FLAGS.num_gpus = 8
+    FLAGS.data_dir = self.train_data_dir
+    FLAGS.vocab_file = self.vocab_file
+    # Sets values directly to avoid validation check.
+    FLAGS['bleu_source'].value = self.bleu_source
+    FLAGS['bleu_ref'].value = self.bleu_ref
+    FLAGS.param_set = 'base'
+    FLAGS.batch_size = 32768
```

Comment:
Let's write this out as 8 * 4096 to make the per gpu batch-size obvious at a glance.  And change the steps to 100K.  Running 100k I reached 27 to 27.2 but never then 27.3 desired.